plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/PrimitiveConfiguratorTest.java

```diff
@@ -20,96 +22,95 @@
  */
 public class PrimitiveConfiguratorTest {
 
-    @Rule
-    public JenkinsRule j = new JenkinsRule();
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
 
     @Rule
     public final EnvironmentVariables environment = new EnvironmentVariables();
 
+    private static ConfiguratorRegistry registry;
+    private static ConfigurationContext context;
+
+    @BeforeClass
+    public static void setup() {
+        registry = ConfiguratorRegistry.get();
+        context = new ConfigurationContext(registry);
+    }
+
     @Test
     public void _boolean() throws Exception {
-        ConfiguratorRegistry registry = ConfiguratorRegistry.get();
-        Configurator c =registry.lookup(boolean.class);
-        final Object value = c.configure(new Scalar("true"), new ConfigurationContext(registry));
+        Configurator c =registry.lookupOrFail(boolean.class);
```

Comment:
Spaces around `c =registry` could be corrected here and in other places.

Reply:
done :smiley: